<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>733</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1113</width>
    <height>733</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1113</width>
    <height>733</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Vista Inserisci Allenatore</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1113</width>
     <height>733</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>InserimentoNuovoAllenatore.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="id">
   <property name="geometry">
    <rect>
     <x>529</x>
     <y>117</y>
     <width>266</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 15px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nome">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>170</y>
     <width>266</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 15px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cognome">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>230</y>
     <width>266</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 15px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="codiceFiscale">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>290</y>
     <width>266</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 15px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataDiNascita">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>350</y>
     <width>266</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 15px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="eta">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>400</y>
     <width>266</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 15px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="telefono">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>460</y>
     <width>266</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 15px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>520</y>
     <width>266</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 15px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="licenza">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>575</y>
     <width>266</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 15px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_conferma">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>630</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 15px;
</string>
   </property>
   <property name="text">
    <string>CONFERMA</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="InserimentoAllenatore.qrc"/>
 </resources>
 <connections/>
</ui>
